lib/shared/permissions.js

```diff
@@ -101,7 +116,22 @@ exports.getEnvironmentCommandPrefix = (environment) => {
     }
 
     if (isWindows) {
-      return [ 'set', `${key}=${value}`, '&&' ];
+      const definition = `${key}=${value}`;
+      return [
+        'set',
+        _.attempt(() => {
+          if (/\s/.test(definition)) {
+            return quoteString(definition);
+          }
+
+          return definition;
+        }),
+        '&&'
+      ];
+    }
+
+    if (/\s/.test(value)) {
```

Comment:
I was under the impression that `sudo-prompt` would pass options to the [ChildProcess](https://nodejs.org/api/child_process.html#child_process_child_process_exec_command_options_callback), but a quick look at the source told me otherwise.
Forking now, and will see if I can throw a PR together that will allow for it.